src/keymanager.ts

```diff
@@ -58,7 +68,13 @@ export class KeyManager {
     return this;
   }
 
-  getAccidental(key) {
+  getAccidental(
```

Comment:
This formatting feels off. Can you put the function and params in one line?

Reply:
you are right, fixed